lib/lightning_web/live/project_live/collaborators.ex

```diff
@@ -76,7 +76,7 @@ defmodule LightningWeb.ProjectLive.Collaborators do
     Enum.map(collaborators, fn collaborator ->
       existing_user =
         Enum.find(existing_users, fn u ->
-          u.email == get_field(collaborator, :email)
+          u.email == String.downcase(get_field(collaborator, :email))
         end)
```

Comment:
This is already part of my PR. Let's not handle it, that way we avoid merge conflicts

Reply:
i've replaced yours with mine so for performance reasons. don't want to call downcase on every email in the instance